docsrc/contributors/fx_converters.rst

```diff
@@ -0,0 +1,133 @@
+.. _conversion:
+
+FX Converters
+==================
+The converter library in Torch-TensorRT is located in ``TensorRT/py/torch_tensorrt/fx/converters``.
+They are categorized into - ``aten_ops_converters``, ``acc_ops_converters`` and ``nn_ops_converters``.
+The individual converters present are useful for the quantization workflow.
+The converters are registered using the ``tensorrt_converter``
+
+Steps
+==================
+
+External Interface
```

Comment:
Rename this `Op Sets`, it doesnt really matter how they are generated (though it is useful to mention this) but the high order bit is that there are different op sets that we could register converters for.